packages/override-tools/src/test/Manifest.test.ts

```diff
@@ -76,7 +76,7 @@ test('ManifestMissingFile', async () => {
 
   const expectedError: ValidationError = {
     type: 'fileMissingFromManifest',
-    file: 'bbb/ccc.win32.js',
+    file: 'bbb\\ccc.win32.js',
```

Comment:
>b\\c [](start = 13, length = 4)

Why here \ is used but in other places / eg line 91

Reply:
I replaced everything with \ in a followup change.

Basically, I used / everywhere initially, but path normalization puts the slashes the other directions on Windows, so I needed to start changing test code when I moved paths internally to be normalized.

It annoys me to no end that Node doesn't have a path object, or path specific compare function, etc. That would be ideal instead of needing to try to converge string representation internally